plugin/pkg/admission/podnodeselector/admission.go

```diff
@@ -114,20 +115,23 @@ func (p *podNodeSelector) Admit(a admission.Attributes) error {
 
 	resource := a.GetResource().GroupResource()
 	pod := a.GetObject().(*api.Pod)
-	namespaceNodeSelector, err := p.getNamespaceNodeSelectorMap(a.GetNamespace())
+
+	var namespaceNodeSelector labels.Set
```

Comment:
nit: you can just use := on the next line rather than the explicit declaration here.  maybe you can fix it up when/if you rebase.  this LGTM.